packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -269,20 +270,28 @@ export const ComposerInput = forwardRef(function ComposerInput(
   const handleKeyDown = useCallback(
     (e: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
       const { key } = e.nativeEvent
+      const isBackspaceAvailable =
+        lastKeyPressMsRef.current === null ||
+        Date.now() - lastKeyPressMsRef.current > 300
```

Comment:
wow, crazy

Reply:
yea its kind of gross, but it works. this was also much easier than handling it by checking the value.